pkg/zone/feature/openclose/group_test.go

```diff
@@ -54,6 +54,11 @@ func Test_mergeOpenClosePositions(t *testing.T) {
 		{"[{10% held},{10% held}]", []value{val(pos(10).Held()), val(pos(10).Held())}, ocp(pos(10).Held()), false},
 		// combine it all together
 		{"[{10% up slow,90% down},{30% up,70% down held}]", []value{val(up(10).Slow(), down(90)), val(up(30), down(70).Held())}, ocp(up(20).Slow(), down(80).Held()), false},
+		// preset tests
+		{"[{open}]", []value{valPres("open")}, ocpPres("open"), false},
+		{"[{nil },{open},{nil }]", []value{valPres(""), valPres("open"), valPres("")}, ocpPres("open"), false},
+		{"[{open},{open},{open}]", []value{valPres(""), valPres("open"), valPres("")}, ocpPres("open"), false},
```

Comment:
This test has the same input as the one above. Looks like it should be `[]value{valPres("open"), valPres("open"), valPres("open")}` to match the name